CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+
+The list of issues assigned to a milestone will give a nice overview of new features and fixed bugs after a release has been done, facilitating the creation of release notes and helping users to know when a fix will be available.
+
+### Issue Labels
+
+Issue labels can serve several purposes:
+
+* Indicate the issue type: `bug`, `enhancement`, `new_feature`, `question`
+* Communicate the current status of an issue: `confirmed`, `help_wanted`
+* Communicate why an issue has not been fixed: `duplicate`, `invalid`, `wontfix`
+* Categorize issues for better overview, e.g. to assign an issue to a specific part of the software. Committers may add categorization labels as required. However, in most cases the `[category]` prefixing in the issue title should be sufficient.
+
+In general we want to keep the number of labels as low as possible in order to avoid contributors to be overwhelmed and to make sure they are actually used. A label is not useful if it's assigned only to a fraction of the issues it belongs to, so it's important for all contributors to use the existing labels consistently.
+
+### Workflow
+
+All committers should feel responsible to read incoming issues. When you read a new issue, please think about an appropriate reaction:
+
+* If possible, assign a type and a status.
+* If appropriate, close the issue and explain why it won't be fixed.
+* If the report seems reasonable, a comment with some feedback to the reporter would be nice (e.g. describe wich action is required next, such as to confirm the issue or find a solution for it), especially if the reporter is not a committer.
+* If you are not familiar enough with the topic of the issue, you might find someone who is and delegate the action.
+
+## Set up your Eclipse Workspace
+Before starting the setup process, make sure that you have both an [Eclipse](https://accounts.eclipse.org/user/register) and a [GitHub](https://github.com/join) account.
+
+1. Download and start the [Eclipse Installer](https://wiki.eclipse.org/Eclipse_Oomph_Installer).
+1. On the initial page, click on the *Switch to Advanced Mode* button in the top right.
+1. On the *Product* page, select *Eclipse IDE for Eclipse Committers*.
+1. On the *Projects* page, select the *Xtext* node.
+1. Choose your preferred installation settings on the *Variables* page. Enter credentials for your Eclipse and GitHub account. If you don't have ssh keys registered with Github then make sure that you select `HTTPS (read-only, anonymous)` or `HTTPS (read-write)` for the Github repository entries.
+1. Finish the wizard, drink a cup of coffee, and watch how your Xtext development environment is assembled.
+
+IMPORTANT: if you had previously set up your workspace when Xtext code was split in several Git repositories, please, start completely from scratch (new installation and new clone).
```

Comment:
I'd leave this out. Existing contributors will likely have followed the repo merge. The avg reader of this document doesn't know about it and it would be confusing at best.

Reply:
Done